src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/update/UpdateTargetLocator.java

```diff
@@ -133,7 +152,39 @@ public UpdateTarget findOrCreate(JsonNode document, String dotPath) {
     throw cantCreatePropertyPath(dotPath, segment, context);
   }
 
-  private String[] splitAndVerify(String dotPath) {
+  /**
+   * Traversal method that is similar to {@link #findIfExists} but that will not return full {@link
+   * UpdateTarget}; instead a non-{@code null} {@link JsonNode} (possibly of type {@code
+   * MissingNode} is returned matching value at given path (or lack thereof in case of {@code
+   * MissingNode}).
+   *
+   * @param document Document on which to evaluate configured path.
+   * @return Value node in given document at configured path, if any; a "missing node" (one for
+   *     which {@code JsonNode.isMissingNode()} returns {@code}).
+   */
+  public JsonNode findValueIn(JsonNode document) {
```

Comment:
@maheshrajamani This would be method called by Sorting logic on given document to fetch a non-null `JsonNode`, compare using `JsonNodeComparator`. Also likely useful for Projection.